doc/lua_api.md

```diff
@@ -8474,6 +8479,9 @@ child will follow movement and rotation of that bone.
           -- Default: false
       }
       ```
+* `get_guid()`: returns a global unique identifier (a string)
+    * For players this is a player name.
+    * For Lua entities, it is a unique generated string.
```

Comment:
* A string which can't be a playername, I suppose?
* Should maybe doc that it's printable.
* Should maybe doc how a lua entity guid looks like, for easier debugging.
* Should doc that this id format will not change between versions, and also not between object reloads, so storing persistently is fine.